app/models/weather.rb

```diff
@@ -22,6 +22,25 @@ def chance_of_rain(day_index)
     raw_day["precipProbability"] * 100
   end
 
+  def precip_type(day_index)
+    type = weather_info["daily"]["data"][day_index]["precipType"]
```

Comment:
I think this method and the one below could have some refactors because they're using/dealing with the same info.